<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>self_set</class>
 <widget class="QDialog" name="self_set">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>自配置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(29, 15, 29);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>730</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>651</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(87, 16, 16);


</string>
   </property>
   <property name="title">
    <string>参数设置</string>
   </property>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>通信最小信号强度：</string>
    </property>
   </widget>
   <widget class="QDial" name="dial_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="value">
     <number>-85</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>(Lr(路损)=alog(d)+b)</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>440</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>覆盖率：</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>560</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>平均SINR最优</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>能效比最高</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>520</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>95</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>路损模型参数：</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>93</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>a=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="PARAMETERA">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/new/prefix1/back.PNG);</string>
         </property>
         <property name="value">
          <double>20.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>b=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="PARAMETERB">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/new/prefix1/back.PNG);</string>
         </property>
         <property name="value">
          <double>32.399999999999999</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>165</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>RSRP0=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="RSRP0">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/back.PNG);</string>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="value">
        <number>-85</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>dBm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 16, 16);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>192</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>基站额定功率=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/back.PNG);</string>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>dBm</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>65</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QSpinBox" name="COVERRATE">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/back.PNG);</string>
       </property>
       <property name="value">
        <number>95</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QDial" name="dial_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>458</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777211</height>
     </size>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>95</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>620</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>恢复默认</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>350</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>临界信噪比：</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>392</y>
      <width>114</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>SINR=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="SINRSPIN">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/back.PNG);</string>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>self_set</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>744</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>self_set</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>750</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RSRP0</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RSRP0</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>COVERRATE</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>COVERRATE</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>COVERRATE</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>COVERRATE</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
